readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
Does it sounds like you can guarantee that the result of such function is always ISTF value or a subtree depending on use case? If yes then its fine from standard perspective I think.